build_projects/dotnet-host-build/StubPackageBuilder.cs

```diff
@@ -82,7 +82,7 @@ private void BuildStubBits(DotNetCli dotnet, string intermediateDirectory)
             projectJson.Append("{");
             projectJson.Append($"  \"version\": \"{version}\",");
             projectJson.Append($"  \"name\": \"{packageId}\",");
-            projectJson.Append("  \"dependencies\": { \"NETStandard.Library\": \"1.5.0-rc2-24008\" },");
+            projectJson.Append("  \"dependencies\": { \"NETStandard.Library\": \"1.5.0-rc2-24027\" },");
```

Comment:
Can we put this into file so automatic `project.json` updates would hit this file too?